en/deploy/README.md

```diff
@@ -291,13 +293,19 @@ We're all done!  Hit the big green **Reload** button and you'll be able to go vi
 
 ## Debugging tips
 
-If you see an error when you try to visit your site, the first place to look for some debugging info is in your **error log** -- you'll find a link to this on the PythonAnywhere web tab. See if there are any error messages in there. The most recent ones are at the bottom. Common problems include
+If you see an error when you try to visit your site, the first place to look for some debugging info is in your **error log** -- you'll find a link to this on the PythonAnywhere Web tab. See if there are any error messages in there; the most recent ones are at the bottom. Common problems include:
```

Comment:
Is there a way to turn "PythonAnywhere Web tab" into a clickable link that takes you to the right place (assuming the reader is logged in)?